deepmd/main.py

```diff
@@ -51,6 +51,18 @@ def get_ll(log_level: str) -> int:
     return int_level
 
 
+def float_list(value_str):
+    """Convert string to a list of float values."""
+    values = value_str.split(",")
+    try:
+        return [float(v) for v in values]
+    except ValueError:
+        raise argparse.ArgumentTypeError(
+            f"'{value_str}' cannot be converted to a list of floats, "
+            f"please use ',' to separate each float value."
+        )
```

Comment:
```suggestion
    except ValueError as e:
        raise argparse.ArgumentTypeError(
            f"'{value_str}' cannot be converted to a list of floats, "
            f"please use ',' to separate each float value."
        ) from e
```

Reply:
We should turn on [`B904`](https://docs.astral.sh/ruff/rules/raise-without-from-inside-except/)